packages/r/raylib/xmake.lua

```diff
@@ -42,6 +42,10 @@ package("raylib")
     end
     add_deps("opengl", {optional = true})
 
+    add_configs("use_external_glfw", {description = "Link raylib against system GLFW instead of embedded one.", default = "OFF", values = {"OFF", "ON", "IF_POSSIBLE"}})
+    add_configs("platform", {description = "Platform to build for.", default = "Desktop", values = {"Desktop", "Web", "Android", "Raspberry Pi"}})
```

Comment:
please remove it, we should use `package:is_plat()` to get platform from `xmake f -p/--plat`